hack/deploy-karmada-agent.sh

```diff
@@ -59,17 +59,7 @@ fi
 export KUBECONFIG="${MEMBER_CLUSTER_KUBECONFIG}" # switch to member cluster
 kubectl config use-context "${MEMBER_CLUSTER_NAME}"
 
-AGENT_IMAGE_PULL_POLICY="Always" # default is 'always' for standalone member cluster to try to use the latest version
```

Comment:
In doubt deleting it

Reply:
We should use `IfNotPresent` as we usually load the image from source not load from the image registry.

I guess we can take this as an environment. 
We can't just delete it as there is a template we have to set:
https://github.com/karmada-io/karmada/blob/a35672271fa0102eb5c8d3e508ca554982834d69/artifacts/agent/karmada-agent.yaml#L25

Reply:
Should I modify like `AGENT_IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY:-IfNotPresent}`

Reply:
That's would be better!